.github/workflows/conformance.yaml

```diff
@@ -640,7 +640,7 @@ jobs:
         uses: kyverno/action-install-chainsaw@d12e54dd3506268b9d5bb61fbbcb2e7eec5fb70c # v0.1.1
       # create cluster
       - name: Create kind cluster and setup Sigstore Scaffolding
-        uses: sigstore/scaffolding/actions/setup@a6e9b1a7a45f10449bbb677c31ce4c1389edd08b
+        uses: sigstore/scaffolding/actions/setup@main
```

Comment:
@vishal-chdhry you need to put a git hash here, not `main`

Reply:
yes, for some reason it is not working, I am trying out main to see if that works. I was using the latest commit hash 🤷

Reply:
Seems to be working for the exact same commit hash on local
```bash
❯ ./hack/setup-kind.sh  --registry-url registry.local:5000 --cluster-suffix cluster.local --k8s-version v1.29.x --knative-version 1.10.0
RUNNING ON Darwin
Running on Darwin
::group:: Install KinD
::endgroup::
::group:: Build KinD Config
apiVersion: kind.x-k8s.io/v1alpha4
kind: Cluster
nodes:
- role: control-plane
  image: "kindest/node:1.29.0@sha256:eaa1450915475849a73a9227b8f201df25e55e268e5d619312131292e324d570"
- role: worker
  image: "kindest/node:1.29.0@sha256:eaa1450915475849a73a9227b8f201df25e55e268e5d619312131292e324d570"

# Configure registry for KinD.
containerdConfigPatches:
- |-
  [plugins."io.containerd.grpc.v1.cri".registry.mirrors."registry.local:5000"]
    endpoint = ["http://registry.local:5000"]

# This is needed in order to support projected volumes with service account tokens.
# See: https://kubernetes.slack.com/archives/CEKK1KTN2/p1600268272383600
kubeadmConfigPatches:
  - |
    apiVersion: kubeadm.k8s.io/v1beta2
    kind: ClusterConfiguration
    metadata:
      name: config
    apiServer:
      extraArgs:
        "service-account-issuer": "https://kubernetes.default.svc.cluster.local"
        "service-account-key-file": "/etc/kubernetes/pki/sa.pub"
        "service-account-signing-key-file": "/etc/kubernetes/pki/sa.key"
        "service-account-api-audiences": "api,spire-server"
        "service-account-jwks-uri": "https://kubernetes.default.svc/openid/v1/jwks"
    networking:
      dnsDomain: "cluster.local"
::endgroup::
::group:: Create KinD Cluster
Creating cluster "kind" ...
⢎⠁ Ensuring node image (kindest/node:1.29.0) 🖼 ^C
❯ git rev-parse HEAD
bea34b6f64147aa09768cd9c4c5d17e71d9e144e
```

Reply:
@vishal-chdhry - can you resolve this? It's blocking the merge.